### PR TITLE
[DOCS] Fix examples

### PR DESCRIPTION
**Pull request by elephaint**
- Adds missing model imports to examples
- Cleans examples from unnecessary imports
- Other tiny fixes to docs

### CODE REVIEW

File: nbs/models.timemixer.ipynb [hunk 1]
```diff
@@ -816,16 +816,8 @@
     "from neuralforecast import NeuralForecast\n",
```
Comment (marcopeix) [inline nbs/models.timemixer.ipynb:816]: <pre class="ql-syntax" spellcheck="false">Line #17.                    max_steps=100,
</pre><p>Could we increase the max step to 500 like in other models? Also gives users an idea that DL should ideally run for around 500, even 1000 steps to get good predictions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: nbs/models.timesnet.ipynb [hunk 2]
```diff
@@ -439,14 +439,12 @@
    "outputs": [],
```
Comment (marcopeix) [inline nbs/models.timesnet.ipynb:439]: <pre class="ql-syntax" spellcheck="false">Line #19.                     #loss=MQLoss(quantiles=[0.2, 0.5, 0.8]),
</pre><p>Let's remove this comments, just for a cleaner documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: nbs/models.tsmixer.ipynb [hunk 3]
```diff
@@ -495,12 +495,11 @@
    "outputs": [],
```
Comment (marcopeix) [inline nbs/models.tsmixer.ipynb:495]: <pre class="ql-syntax" spellcheck="false">Line #21.                    max_steps=200,
</pre><p>Same here, I would do at least 500, just to better guide users.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

File: nbs/models.tsmixerx.ipynb [hunk 4]
```diff
@@ -658,12 +658,11 @@
    "outputs": [],
```
Comment (marcopeix) [inline nbs/models.tsmixerx.ipynb:658]: <pre class="ql-syntax" spellcheck="false">Line #22.                    max_steps=200,
</pre><p>Again, I would use 500.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

File: nbs/models.timemixer.ipynb (hunk 1 above)
Comment (elephaint) [reply nbs/models.timemixer.ipynb:816]: Thanks, changed to 500

File: nbs/models.tsmixerx.ipynb (hunk 4 above)
Comment (elephaint) [reply nbs/models.tsmixerx.ipynb:658]: Changed to 500

File: nbs/models.tsmixer.ipynb (hunk 3 above)
Comment (elephaint) [reply nbs/models.tsmixer.ipynb:495]: Changed to 500

**Review by marcopeix — APPROVED** [review]

**Pull request merged**